web/api/js/codechecker-api-js/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "codechecker-api-js",
+  "version": "6.24.0",
+  "description": "Generated jQuery compatible API stubs for CodeChecker server.",
+  "main": "lib/codechecker-api-js",
+  "homepage": "https://github.com/Ericsson/codechecker",
+  "readmeFilename": "README.md",
+  "bugs": {
+    "url": "https://github.com/Ericsson/codechecker/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Ericsson/codechecker.git",
```

Do we need `git+` in the url?

```suggestion
    "url": "https://github.com/Ericsson/codechecker.git",
```

---

I've seen multiple examples where it was there, but I did not found any official documentation if it is required or not.